lib/collection/src/shards/local_shard.rs

```diff
@@ -738,6 +740,13 @@ impl LocalShard {
             payload_schema: schema,
         }
     }
+
+    pub async fn schedule_preheat_disk_cache_tasks(&self) {
```

Comment:
I don't want to nitpick, but on second thought, this is not exactly 'preheating disk cache'.

Maybe we should call this `load_mmap` or `prime_mmap`. Or even the technical term: `prefault_mmap_pages`.

Or do you think the current name makes more sense?

Reply:
Tbh, I was not happy with the name either. I like the "prefault" though!